src/docs/common_tasks/multiple_jvm_versions.md

```diff
@@ -0,0 +1,53 @@
+
+## Problem
+
+You want to compile and/or run different test or binary targets using different JVMs.
+
+Or, you need to specify command-line options globally for the JVM on a per-JVM version basis. For
+example, using `--add-opens` flags, but only when running on JVMs that support it.
+
+## Solution
+
+Declare JVM platforms using the `jvm-platform` configuration section to define the set of available
+platforms.
+
+On targets, add the kwarg `runtime_platform` set to the name of one of the platforms defined in
+`jvm-platform`. That will tell pants to use that platform when running that target in a JVM.
```

Comment:
It may be good to clarify here if you need to add it for _every_ target, or if not, what the default is.